Tests/web3swiftTests/localTests/EIP1559BlockTests.swift

```diff
@@ -93,16 +93,16 @@ class EIP1559BlockTests: LocalTestCase {
                                 totalDifficulty: uselessBlockPart.totalDifficulty,
                                 extraData: uselessBlockPart.extraData,
                                 size: uselessBlockPart.size,
-                                gasLimit: touple.currentGasLimit,
-                                gasUsed: touple.currentGasLimit / 2,
+                                gasLimit: tuple.currentGasLimit,
+                                gasUsed: tuple.currentGasLimit / 2,
                                 baseFeePerGas: Web3.InitialBaseFee,
                                 timestamp: uselessBlockPart.timestamp,
                                 transactions: uselessBlockPart.transactions,
                                 uncles: uselessBlockPart.uncles)
 
-            if touple.is1559 {
+            if tuple.is1559 {
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.